templates/rustdoc/head.html

```diff
@@ -3,8 +3,12 @@
             media="all" />
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/pure/0.6.0/grids-min.css" type="text/css"
             media="all" />
+<<<<<<< HEAD
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css"
             type="text/css" media="all" />
+=======
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css" />
```

Comment:
Where does this URL come from? I don't see anywhere on the main website listing it as a CDN?

Reply:
[This page](https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4) in the codeblock of the `Upgrading Manually` section